pocs/utils/images/__init__.py

```diff
@@ -92,7 +95,39 @@ def _make_pretty_from_fits(fname, **kwargs):
     new_filename = fname.replace('.fits', '.jpg')
 
     data = getdata(fname)
-    plt.imshow(data, cmap='cubehelix_r', origin='lower')
+
+    norm = ImageNormalize(interval=MinMaxInterval(), stretch=LogStretch())
+
+    wcs = False
+
+    """
+    try:
```

Comment:
```
try:
    w = wcs.WCS(filename)
    assert w.is_celestial
except AssertionError:
    pass
```

Reply:
Or just skip the `try` and use `if wcs.is_celstial` below.

Reply:
Get this error message `AttributeError: 'bool' object has no attribute 'WCS'` for `w = wcs.WCS(filename)`.

Reply:
@jermainegug That's because in your code you're setting `wcs` to `False`, whereas @wtgee's code suggestion is assuming that `wcs` is `astropy.wcs` (i.e. you're doing `from astropy import wcs`)

Reply:
Yes, I didn't adjust it for your code, I just copied what we have elsewhere.

You import `WCS` directly so use that instead of `wcs.WCS`.

But, you are also mixing types. You create `wcs = False` (a boolean) up above and then assign `wcs=WCS(filename)` (a `WCS` class) to it. I would just skip the `try` and use the `is_celestial` in the `if` statement directly after getting the wcs.

Reply:
What is `wcs` in this case? `wcs=WCS(fname)`?

Reply:
R.T.D.

It's an `WCS` object, i.e. an instance of the `astropy.wcs.WCS` class.

http://docs.astropy.org/en/stable/api/astropy.wcs.WCS.html#astropy.wcs.WCS